MoviesMVVM/MoviesMVVM/Enums/MovieType.swift

```diff
@@ -3,12 +3,23 @@
 
 import Foundation
 
-/// Энам типов
-enum MovieType: Int {
-    // Тип "Популярный"
-    case popular = 0
-    // Тип "С высоким рейтингом"
-    case topRated = 1
-    // Тип "Скоро"
-    case upComing = 2
+/// Энам категорий фильмов
+enum MovieType: String, CaseIterable {
+    /// Тип "Популярный"
+    case popular
+    /// Тип "С высоким рейтингом"
+    case topRated
+    /// Тип "Скоро"
```

Comment:
```suggestion
    /// "Скоро"
```
что за "Тип" выглядит странно